streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -1141,17 +1134,17 @@ StreamThread updateThreadMetadata(final String adminClientId) {
         return this;
     }
 
-    private void updateThreadMetadata(final Map<TaskId, StreamTask> activeTasks,
-                                      final Map<TaskId, StandbyTask> standbyTasks) {
+    private void updateThreadMetadata(final Map<TaskId, Task> activeTasks,
+                                      final Map<TaskId, Task> standbyTasks) {
```

Comment:
Why do we weaken the types here?

Reply:
Because the task manager now maintains the active / standby tasks in a single collection so the exposed APIs only return a map of `Tasks`. Also for the caller there's no need to have the stronger types.